StoreApp.BLL/Services/ProductService.cs

```diff
@@ -58,6 +65,49 @@ public IEnumerable<ProductDTO> GetAll()
             return config.Map<IEnumerable<Product>, IEnumerable<ProductDTO>>(DataBase.Products.GetAll());
         }
 
+        public bool ValidateNewProduct(ProductDTO product)
+        {
+            var validate = DataBase.Products.Find(p => p.Name.Equals(product.Name) &&
+                                                       p.BrandId.Equals(product.BrandId) &&
+                                                       p.ProducerId.Equals(product.ProducerId));
+
+            foreach (var item in validate)
+            {
+                return false;
+            }
+
+            return true;
```

Comment:
I don't understand this approach. Need to discuss it

Reply:
fixed